store/searchlayer/user_layer.go

```diff
@@ -146,45 +145,48 @@ func (s *SearchUserStore) autocompleteUsersInChannelByEngine(engine searchengine
 	return autocomplete, nil
 }
 
+// We're contemplating different scenarios in this method:
```

Comment:
In godoc is a good practice to use the name of the function as the beginning of the explanation.